packages/ia-topnav/src/primary-nav.js

```diff
@@ -139,18 +145,30 @@ class PrimaryNav extends TrackedElement {
     `;
   }
 
+  get secondLogoSlot() {
+    return this.allowSecondaryIcon
+      ? html`<slot name="opt-sec-logo" data-event-click-tracking="${this.config.eventCategory}|${this.secondIdentitySlot}"><slot>`
```

Comment:
Q: Does this click tracking work? I was expecting the link inside it to capture the click.

Reply:
Yano, idk! this was a hopeful add-on ^_^

this `data-` attribute connects to the 1% GA instance, I can add a click handler to see if it reacts to a click event...

Reply:
yah, no dice. yanking.